Ladybird/AppKit/UI/LadybirdWebView.mm

```diff
@@ -527,6 +527,18 @@ - (void)setWebViewCallbacks
                      returnCode:NSModalResponseCancel];
     };
 
+    m_web_view_bridge->on_request_color_picker = [self](Color current_color) {
+        NSColorPanel* panel = [NSColorPanel sharedColorPanel];
+        [panel setColor:Ladybird::gfx_color_to_ns_color(current_color)];
+        [panel setShowsAlpha:NO];
+        NSNotificationCenter* nc = [NSNotificationCenter defaultCenter];
```

Comment:
`auto* notification_center` (a newline above this would be nice IMO, but no big deal).

Reply:
Also, why are we listening for the window to close instead of using `NSColorPanel setAction`? At least in firefox, input handlers are updated without waiting for the window to close.

Reply:
Sending live color updates to the webview is the better, but the API currently supports only one color update message with `color_picker_closed` that also signals that the color picker is closed.

Reply:
Got it, that is unfortunate, but we can update that in the future.